dep_checker/npm_audit.py

```
@@ -39,126 +72,154 @@ class AuditParseError(Exception):
+                package_json_files.append(package_json)
+
             logger.info(f"Found {len(package_json_files)} package.json files in deps/ folder")
             if excluded_count > 0:
-                logger.info(f"Excluded {excluded_count} package.json files based on exclusion list: {self.exclude_paths}")
-            
+                logger.info(
+                    f"Excluded {excluded_count} package.json files based on exclusion list: {self.exclude_paths}"
+                )
+
             return package_json_files
         except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)